index.js

```diff
@@ -22,3 +22,5 @@ Object.defineProperty(exports, 'test', {
 });
 
 exports.Transaction = require('loopback-connector').Transaction;
```

Comment:
Oh, never mind. I was confused. The export is in juggler. We need to keep Transaction as it.